bucket/confuser-ex.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "1.5.0",
+    "description": "An open-source protector for .NET applications",
```

Comment:
What does "protector" mean?? For end user it is completely irrelevant description.

Reply:
It's an IL obsfucator